wchamber/sam/kustomization.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  env: wchamber
+namespace: wchamber
+patchesStrategicMerge:
+- istio-service.yaml
+resources:
+# Will get overwritten with correct version on deploy:
+- github.com/databiosphere/sam//k8s-config?ref=0.0.0
```

sam is in the broadinstitute org
```suggestion
- github.com/broadinstitute/sam//k8s-config?ref=0.0.0
```